src/db/models.py

```diff
@@ -1,48 +1,101 @@
 from . import connection
```

Comment:
Can we split these classes into different files? This isn't a scalable way to define the models

Reply:
yea